dizoo/atari/config/serial/pong/pong_sqil_config.py

```diff
@@ -44,11 +44,11 @@
         type='atari',
         import_names=['dizoo.atari.envs.atari_env'],
     ),
-    env_manager=dict(type='subprocess'),
+    env_manager=dict(type='base', force_reproducibility=True),
```

Comment:
we have change `force_reproducibility` to `reset_inplace`, add don't add other arguments in create_cfg except `type` and `import_names`

Reply:
thx！ corrected